src/Sulu/Bundle/AdminBundle/Resources/js/containers/ViewRenderer/ViewRenderer.js

```diff
@@ -48,37 +49,36 @@ class ViewRenderer extends React.Component<Props> {
         }
     }
 
-    getView = (route: Route): View => {
-        const View = viewRegistry.get(route.type);
-
-        if (!View) {
-            throw new Error('View "' + route.type + '" has not been found');
-        }
```

Comment:
This is unnecessary check which was removed because the `viewRegistry.get` already check and throw an exception is view does not exist there was no need todo the same in the `viewRenderer`.